lib/logstash/outputs/elasticsearch_java.rb

```diff
@@ -238,14 +228,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # in situations where you cannot permit connections outbound from the
   # Elasticsearch cluster to this Logstash server.
   #
-  # The 'http' protocol will use the Elasticsearch REST/HTTP interface to talk
-  # to elasticsearch.
-  #
   # All protocols will use bulk requests when talking to Elasticsearch.
   #
   # The default `protocol` setting under java/jruby is "node". The default
   # `protocol` on non-java rubies is "http"
-  config :protocol, :validate => [ "node", "transport", "http" ]
+  config :protocol, :validate => [ "node", "transport"], :default => "node"
```

Comment:
+1 default to transport. We require (basically) hosts anyway and further that the main benefit of node for logstash (by intention, anyway) was multicast discovery which is disabled in 2.0